charts/k8spacket/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if and .Values.k8sPacket.tcp.metrics.enabled .Values.k8sPacket.tcp.metrics.serviceMonitor.enabled }}
```

Comment:
There is another value for metrics for TLS events (see: https://github.com/k8spacket/k8spacket-helm-chart/blob/master/charts/k8spacket/values.yaml#L93)

I think it should be here like:
```
(tcp.metrics.enabled || tls.metrics.enabled) && serviceMonitor.enabled